includes/Admin/Dashboard.php

```diff
@@ -303,6 +303,12 @@ public function load_stories_dashboard(): void {
 			'/web-stories/v1/settings/',
 			'/web-stories/v1/publisher-logos/',
 			'/web-stories/v1/users/me/',
+			'/web-stories/v1/taxonomies/?' . build_query(
+				[
+					'type'    => $this->story_post_type->get_slug(),
+					'context' => 'edit',
```

Comment:
```suggestion
					'context' => 'edit',
					'hierarchical' => 'true'
```

Need to add this now we passing this param in the quest to ensure that is correctly preloaded.

Reply:
Heads up that the arrows need to be aligned, otherwise there's a lint error.